content/blog/2018-05-02-onboarding-is-work.md

```diff
@@ -0,0 +1,325 @@
+
+``` r
+namespace_ro <- readr::read_csv("output/namespace.csv")
+namespace_ro <- dplyr::left_join(namespace_ro, airtable, by = "package")
+ggplot(namespace_ro) +
+  geom_point(aes(exports, review_hours)) +
+  hrbrthemes::theme_ipsum(base_size = 16,
+                          axis_title_size = 16) +
+  ylab("Reviewing time (hours)") +
+  xlab("No. of exports")
+```
+
+![](/img/blog-images/2018-05-02-onboarding-is-work/scatterplot-size-vs-reviewing-time-1.png)
+
+There are a few potential explanations to this that might be fruitful to
+explore. For instance, does this mean that there’s only so much time,
+and so larger packages get less scrutiny per line of code? Or does
+review time just depend more on the reviewer than the package?
+
+### Work done by editors
+
+Editors manage the review process, performing initial package checks,
+identifying and contacting reviewers, and then moderating the cajoling
+the process forward. Our best measure for editor effort is the number of
+packages handled by an editor in a given time frame, which we can track
+from the assignments on GitHub issues since the start of onboarding.
+Looking at this over time, we can see how editor workloads changed in
+response to growing number of assignments and how we have attempted to
+manage this by expanding our editorial board:
+
+``` r
+library(tidyverse)
+library(gh)
+library(lubridate)
+
+issues <- gh("/repos/ropensci/onboarding/issues?state=all&labels=package", .limit=1000)
+
+edits = map_df(issues,
+~data_frame(url = .$html_url,
+editor = .$assignee$login %||% NA_character_,
+opened = as.Date(.$created_at))) %>%
+filter(!is.na(editor)) %>%
+mutate(quarter = paste(year(opened), quarter(opened), sep="Q"),
+half =  paste(year(opened), if_else(quarter(opened) <= 2, 1, 2), sep="H"),
+year = year(opened))
+
+edits %>%
+group_by(editor, half) %>%
+summarize(n_assigned = n()) %>%
+{ full_join(., crossing(editor = unique(.$editor),  #can't get expand() to work.
+half = unique(.$half))) } %>%
+mutate(n_assigned = coalesce(n_assigned, 0L)) %>%
+ggplot(aes(x=half, y=n_assigned, fill=editor)) +
+geom_col(position="dodge") +
+geom_hline(yintercept = c(3, 6)) +
+xlab("Half / Year") + ylab("No. Issues Handled")+
+    hrbrthemes::theme_ipsum() +
+  theme(legend.position = "bottom") +
+  viridis::scale_fill_viridis(discrete = TRUE)
+```
+
+![number of assignments per editor per half a
+year](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-7-1.png)
+
+### Outlook: decreasing work by automation
+
+As mentioned [in this blog post about the authors and reviewers
+survey](https://ropensci.org/blog/2018/04/17/author-survey/), we’re in
+the process of trying to maximize automation of all than can be
+automated in order to reduce and simplify work for everyone involved.
+This way, humans can focus on what they’re best at. Our current
+automation efforts include two packages in development: one for package
+authors, [`rodev`](https://github.com/ropenscilabs/rodev), and one for
```

Comment:
Should you mention `rodev` is by rOpenSci staff since Anna is identified? Maybe "...by rOpenSci staff and `pkgreviewr` by community member, Anna Krystalli"